api/src/test/groovy/com/okta/idx/sdk/api/client/IDXAuthenticationWrapperTest.groovy

```diff
@@ -355,6 +355,38 @@ class IDXAuthenticationWrapperTest {
         assertThat(authenticationResponse.getTokenResponse().getIdToken(), notNullValue())
     }
 
+    @Test
+    void beginTransactionWithAndWithoutActivationTokenTest() {
```

Comment:
Is this name right? It says without activation token. But it seems like it’s with activation token.

Reply:
yes, it tests both [with](https://github.com/okta/okta-idx-java/blob/rc_2.0.0/api/src/test/groovy/com/okta/idx/sdk/api/client/IDXAuthenticationWrapperTest.groovy#L372) and [without](https://github.com/okta/okta-idx-java/blob/rc_2.0.0/api/src/test/groovy/com/okta/idx/sdk/api/client/IDXAuthenticationWrapperTest.groovy#L383) activation token.